### PR TITLE
Refactor notification service's job planner events table layout...

### PR DESCRIPTION
**Pull request by AlexFalessi**
and link notification service user guide + admin guide in JP doc

### CODE REVIEW

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc [hunk 1]
```diff
@@ -207,7 +207,8 @@ An inclusion or exclusion calendar can become not retrievable if it cannot be do
 
 == Notifications
 
-The <<_glossary_job_planner,*Job Planner*>> can send notifications to users when particular events occur. The <<_glossary_notification_service,*Notification service*>> documentation can be found link:../admin/ProActiveAdminGuide.html#_notification_service[here].
+The <<_glossary_job_planner,*Job Planner*>> can send notifications to users when particular events occur. The <<_glossary_notification_service,*Notification service*>> user guide documentation can be found link:../user/ProActiveUserGuide.html#_notification_service[here] and it's
```
Comment (mboussaa) [inline src/docs/JobPlanner/references/JobPlannerServiceReference.adoc:210]: it's => it is
Always use formal form

Comment (mboussaa) [reply src/docs/JobPlanner/references/JobPlannerServiceReference.adoc:210]: ah is it "its" ? I

Comment (mboussaa) [reply src/docs/JobPlanner/references/JobPlannerServiceReference.adoc:210]: can be found is available ...

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc [hunk 2]
```diff
@@ -207,7 +207,8 @@ An inclusion or exclusion calendar can become not retrievable if it cannot be do
...
+configuration in the admin guide link:../admin/ProActiveAdminGuide.html#_notification_service[here].
```
Comment (mboussaa) [inline src/docs/JobPlanner/references/JobPlannerServiceReference.adoc:211]: same

File: src/docs/user/references/Notification-service.adoc [hunk 3]
```diff
@@ -353,22 +353,16 @@ INFO, WARNING, ERROR, CRITICAL. The default severity is INFO.
 [cols="2,2,2,5,5"]
 |===
 | *Type* | *Readable name* | *Severity* | *Message* | *Description*
-| `ASSOCIATION_FAILED`
-| Association failed
+.3+| `ASSOCIATION_FAILED`
+.3+| Association failed
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc:357]: what is .3+ ? and why we have .2+ later?

File: src/docs/user/references/Notification-service.adoc [hunk 4]
```diff
@@ -353,22 +353,16 @@ INFO, WARNING, ERROR, CRITICAL. The default severity is INFO.
...
 located \https://{HOST}/catalog/buckets/basic-examples/resources/Native_Task_Windows/raw.
 Changing the status of the association 152 of the calendar every_minute to FAILED.
 | A <<_glossary_catalog, *Catalog*>> resource is not found, the association changes to FAILED status
-| `ASSOCIATION_FAILED`
-| Association failed
-| ERROR
-| The association 52 of the workflow Variables_Propagation to the calendar every_monday had a failed execution
-, as it is configured, the association status is set to FAILED
+.2+| ERROR
+| The association 52 of the workflow Variables_Propagation to the calendar every_monday had a failed execution, as it is configured, the association status is set to FAILED
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc:364]: , as it is configured => As it is configured,
even though we could not probably edit this probably

**Review by mboussaa — APPROVED** [review]

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc (hunk 1 above)
Comment (AlexFalessi) [reply src/docs/JobPlanner/references/JobPlannerServiceReference.adoc:210]: Applied it

File: src/docs/user/references/Notification-service.adoc (hunk 3 above)
Comment (AlexFalessi) [reply src/docs/user/references/Notification-service.adoc:357]: This is to merge cells in tables like this

image

File: src/docs/user/references/Notification-service.adoc (hunk 4 above)
Comment (AlexFalessi) [reply src/docs/user/references/Notification-service.adoc:364]: Absolutely This needs to be changed in job planner service

**Review by marounkoussaifi — APPROVED** [review]

**Comment by AlexFalessi** [comment]
Retest this please

**Pull request merged**